src/tests/vcs.test.js

```diff
@@ -20,37 +20,37 @@ describe("determineProvider Returns as expected", () => {
   test("Returns unknown VCS Object, when unkown is passed", () => {
     const tmp = "https://new-vcs.com/pulsar-edit/pulsar";
     const res = vcs.determineProvider(tmp);
-    expect(res.type).toBe("na");
-    expect(res.url).toBe(tmp.toLowerCase());
+    expect(res.type).toBe("unkown");
```

Comment:
```suggestion
    expect(res.type).toBe("unknown");
```

I assume you have a typo here

Reply:
Thanks for finding these, would hate to put a whole bunch of typos into the database like this